docs/tutorials/q32d14.ipynb

```diff
@@ -0,0 +1,3360 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Copyright 2020 Google"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "# https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/qsim/blob/master/docs/tutorials/q32d14.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/quantumlib/qsim/blob/master/docs/tutorials/q32d14.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/qsim/docs/tutorials/q32d14.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "RWNGOoXe3xdI"
+   },
+   "source": [
+    "# Simulate large quantum circuit\n",
+    "\n",
+    "\n",
+    "The image below describes the circuit built in code in following code cells. This is a large circuit, with 32 qubits and a gate depth of 14.\n",
+    "\n",
+    "## IMPORTANT\n",
+    "Due to the size of this circuit, it will not run on a default colab kernel, and it is unlikely to run on your personal computer. We recommend following the tutorial provided here [Quantum simulation on GCP with _Cirq_ and _qsim_](https://github.com/quantumlib/qsim/tree/master/docs/tutorials/qsimcirq_gcp.md). This describes how to run on a larger Google Cloud VM.\n",
```

Comment:
```suggestion
    "Caution: Due to the size of this circuit, it will not run on a default Colab kernel and it is unlikely to run on your personal computer. Follow the [Quantum simulation on GCP](https://github.com/quantumlib/qsim/tree/master/docs/tutorials/qsimcirq_gcp.md) tutorial to run on a larger Google Cloud VM.\n",
```

Reply:
Done.